pkg/kubelet/kubelet_test.go

```diff
@@ -293,7 +301,7 @@ func TestKillContainerWithError(t *testing.T) {
 		Err:           fmt.Errorf("sample error"),
 		ContainerList: append([]docker.APIContainers{}, containers...),
```

Comment:
I know this line is not part of this PR, but why write `append([]docker.APIContainers{}, containers...)` when you could write `containers` ?